lib/sls-api-file-delete-error.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+class SlsApiFilDeleteError extends Error {
```

Comment:
```suggestion
class SlsApiFileDeleteError extends Error {
```

Y las demás veces en el archivo